src/System.Net.Primitives/src/MatchingRefApiCompatBaseline.uap.txt

```diff
@@ -0,0 +1,12 @@
+Compat issues with assembly System.Net.Primitives:
+MembersMustExist : Member 'System.Boolean System.Net.Cookie.IsQuotedDomain' does not exist in the implementation but it does exist in the contract.
```

Comment:
@davidsh can you please confirm whether these should be exposed in the reference assembly?

Reply:
Please just baseline this for now.